lib/src/text-input/TextInput.jsx

```diff
@@ -889,20 +880,22 @@ const Error = styled.span`
 `;
 
 const Suggestions = styled.ul`
-  background-color: ${(props) => (props.isError ? props.theme.errorMessageBackgroundColor : "#ffffff")};
   position: absolute;
   z-index: 1;
-  max-height: 160px;
-  overflow: auto;
+  max-height: 304px;
+  overflow-x: auto;
```

Comment:
Why do we have an auto `overflow-x` property value? Shouldn't be `overflow-y:scroll;`?

Reply:
You are right Alex, changed to `overflow-y: auto`, thanks!